### PR TITLE
19 alt padding issue in tarotspreadcontainer

### PR DESCRIPTION
**Pull request by arthvadrr**
# Pull Request Title

**🐾Description of Purrrrr Request🐾**

### Type of Change

- [ 🐈 ] Mew Feature added
- [ 🐈 ] Bug fixedcard
- [ 🐈 ] Re-fur-ctor / Cleanup

### Description of Change

- Refactored tarot card spreads to dynamic layouts (and made some other style changes)
- New tarot spread list component

### Screenshots or Demo (optional)

cards

### Checklist
- [ 🐈 ] Follows coding standards
- [ 🐈 ] Ready fur review

### CODE REVIEW

**Comment by arthvadrr** [comment]
@madeleine-patience I've finished my changes to the tarotSpread. Please review the changes when you can and lmk what you think. I added descriptions to the layout properties so it should be a bit easier to follow, and I mimicked the original transitions/styles. We can go over it sometime soon.

If you're happy with it we can merge into your #19 or into main.

Also a heads up, it looks like vercel-bot is trying to deploy on _every commit_, looks broken. I'm going to stop committing to the project for now - I'm unsure if I should with the failed vercel checks.

**Review by madeleine-patience — COMMENTED** [review]
image


My only initial concern is I really don't like how the cards are moving outside of the box, I feel like they should be contained within the container visually, i think the animations should be smaller and subtler.

File: components/tarotSpreadSelectionBox/TarotSpreadSelectionBox.tsx [hunk 1]
```diff
@@ -1,224 +1,123 @@
...
-          display='flex'
-          alignItems='center'
+          display='grid'
           justifyContent='center'
+          gridTemplateColumns={
+            isHovered && layout.gridHover
+              ? `${layout.gridHover} 0fr`
+              : `${layout.gridTemplateColumns || 'auto'} 0fr`
+          }
+          sx={{
```
Comment (madeleine-patience) [inline components/tarotSpreadSelectionBox/TarotSpreadSelectionBox.tsx:79]: Maybe this is a preference thing but if we have some styles should we put everything inside of the SX?

File: components/tarotSpreadSelectionBox/TarotSpreadSelectionBox.tsx [hunk 2]
```diff
@@ -1,224 +1,123 @@
...
         <Typography
-          variant='body1'
-          textAlign='center'
-          fontSize='bold'
-          fontWeight={600}
-          color={palette.pinks.dark}
+          variant='body2'
+          p={4}
+          pt={0}
+          sx={{
```
Comment (madeleine-patience) [inline components/tarotSpreadSelectionBox/TarotSpreadSelectionBox.tsx:115]: Same here about SX

**Review by madeleine-patience — APPROVED** [review]

**Pull request merged**